artscii/src/tests/DisplayManger.test.js

```diff
@@ -0,0 +1,49 @@
+import { render } from "@testing-library/react";
+import DisplayManager from '../components/displayManager/DisplayManager'
+import { validateDisplayManagerProps } from '../components/displayManager/validateDisplayManagerProps';
+
+// Mocking validateDisplayManagerProps as that is not the SUT
+jest.mock('../components/displayManager/validateDisplayManagerProps', () => ({ validateDisplayManagerProps: jest.fn() }))
+console.error = jest.fn()   // silence console.error() for testing
+let testDisplayManager;
+
+
+beforeEach(() => {
+    testDisplayManager = <DisplayManager src='abc' displayMode='image'/>
+})
+
+
+// Tests begin
+test("renders DisplayManager component", () => {
+    render(
+        testDisplayManager
+    );
+    const displayManagerElement = document.querySelector(".display-window");

```

Comment:
I ran into the same lint warning while making the app test. Seems like direct access to DOM is frowned down upon. I think the proper way of doing it would be to import 'screen' from '@testing-library/react' and you would have something like `const displayManagerElement = screen.getByRole("display-window");`, but that would mean we would have to add roles to our components. It doesn't seem like using direct access to DOM invalidates the testing but might make it more brittle and less maintainable. However, our program's scope is pretty small so I'm not sure if it's worth it. If you end up making this change I'll make a ticket to update the App testing for consistency.

Reply:
Otherwise, great job! All the tests seem to have good coverage